### PR TITLE
[DDCE-3097] Dependency Upgrades

### PR DESCRIPTION
**Pull request by Nboaram**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -5,14 +5,14 @@ import play.core.PlayVersion.current
 object AppDependencies {
 
   lazy val compile = Seq(
-    "uk.gov.hmrc"          %% "bootstrap-backend-play-28"     % "5.14.0",
-    "uk.gov.hmrc"          %% "crypto"                        % "6.0.0",
+    "uk.gov.hmrc"          %% "bootstrap-backend-play-28"     % "5.24.0",
```
Comment (frikit) [inline project/AppDependencies.scala:8]: could be 6.2

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -23,20 +23,19 @@ object AppDependencies {
...
-    "org.jsoup"              % "jsoup"                     % "1.14.2",
+    "org.mockito"            % "mockito-core"              % "4.6.1",
+    "org.jsoup"              % "jsoup"                     % "1.15.2",
     "org.pegdown"            % "pegdown"                   % "1.6.0" ,
     "org.scalatest"          %% "scalatest"                % "3.2.9" ,
     "org.scalatestplus.play" %% "scalatestplus-play"       % "5.1.0" ,
     "org.scalatestplus"      %% "mockito-3-4"              % "3.2.9.0",
     "com.vladsch.flexmark"   % "flexmark-all"              % "0.35.10",
-    "org.scalacheck"         %% "scalacheck"               % "1.15.4",
-//    "uk.gov.hmrc"            %% "http-verbs-test"          % "1.8.0-play-27",
```
Comment (frikit) [inline project/AppDependencies.scala:34]: remove comment?

File: project/build.properties [hunk 3]
```diff
@@ -1 +1 @@
-sbt.version=1.4.9
+sbt.version=1.5.8
```
Comment (frikit) [inline project/build.properties:1]: could be 1.6

**Review by frikit — CHANGES_REQUESTED** [review]

File: project/AppDependencies.scala (hunk 2 above)
Comment (Nboaram) [reply project/AppDependencies.scala:34]: Oh I see, you've highlighted my deletion of it.

File: project/AppDependencies.scala [hunk 4]
```diff
@@ -5,14 +5,14 @@ import play.core.PlayVersion.current
...
 
   lazy val compile = Seq(
-    "uk.gov.hmrc"          %% "bootstrap-backend-play-28"     % "5.14.0",
-    "uk.gov.hmrc"          %% "crypto"                        % "6.0.0",
+    "uk.gov.hmrc"          %% "bootstrap-backend-play-28"     % "6.2.0",
+    "uk.gov.hmrc"          %% "crypto"                        % "6.1.0",
     "uk.gov.hmrc"          %% "play-json-union-formatter"     % "1.15.0-play-28",
-    "uk.gov.hmrc"          %% "simple-reactivemongo"          % "7.31.0-play-27",
-    "uk.gov.hmrc"          %% "mongo-lock"                    % "7.0.0-play-27",
+    "uk.gov.hmrc"          %% "simple-reactivemongo"          % "8.1.0-play-27",
```
Comment (frikit) [inline project/AppDependencies.scala:11]: we are on play 28 maybe we need to upgrade this to play 28?

**Review by frikit — CHANGES_REQUESTED** [review]

File: project/plugins.sbt [hunk 5]
```diff
@@ -2,14 +2,14 @@ resolvers += "HMRC-open-artefacts-maven" at "https://open.artefacts.tax.service.
 resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 resolvers += Resolver.typesafeRepo("releases")
 
-addSbtPlugin("com.typesafe.play"         % "sbt-plugin"             % "2.8.8")
+addSbtPlugin("com.typesafe.play"         % "sbt-plugin"             % "2.8.15")
```
Comment (Nboaram) [inline project/plugins.sbt:5]: this can go to 2.8.16

**Pull request merged**